packages/web-frontend/src/components/View/MatrixWrapper/styles/getStyles.js

```diff
@@ -298,6 +298,20 @@ const getStyles = (isPrintMode, cellCharacters) => {
       position: 'absolute',
       top: 20,
     },
+    footer: {
```

Comment:
This seems like a strange pattern, to calculate the style high in the hierarchy tree rather than passing through the information required to calculate it (in this case none) close to where they are rendered, but given it is the existing pattern it makes sense to use it...

Reply:
Yeah was strange to me too, went with following existing code for consistency